cs-lists/src/main/java/io/cloudslang/content/utils/ListProcessor.java

```diff
@@ -269,5 +281,67 @@ private static double roundTo(double value, int decimals) {
         }
         return list.split(delimiter);
     }
-}
 
+    public static void listContainsElements(String subList, String container, String delimiter, Map<String, String> result, String ignoreCase) {
```

Comment:
I would rather see this method signature as: 
public static boolean listContainsElements(String subList, String container, String delimiter, String ignoreCase) throws Exception

Depending on the boolean returned by this method, you populate the map entries. If Exception is thrown,  populate fields accordingly.
I see this approach more natural